<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prefs_projectmain</class>
 <widget class="QWidget" name="prefs_projectmain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>340</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="l2">
     <property name="text">
      <string comment="@label:textbox">Glossary:</string>
     </property>
     <property name="buddy">
      <cstring>glossaryTbx</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="LangCodeSaver" name="kcfg_LangCode"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="l0">
     <property name="text">
      <string comment="@label:textbox">Target language:</string>
     </property>
     <property name="buddy">
      <cstring>LangCode</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="kcfg_MailingList"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="kcfg_ProjectID">
     <property name="toolTip">
      <string>Project ID is used to differentiate translation memories of absolutely different projects and languages.

For example, if you have several projects for translating KDE applications
(e.g. they are in different repositories), use same ID for all of them.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="KComboBox" name="LangCode">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Target language of the project.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="KUrlRequester" name="glossaryTbx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="l4">
     <property name="text">
      <string comment="@label:textbox">Mailing list:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_MailingList</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="l3">
     <property name="text">
      <string comment="@label:textbox">ID:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_ProjectID</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="RelPathSaver" name="kcfg_GlossaryTbx"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="KUrlRequester" name="poBaseDir">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>16</height>
      </size>
     </property>
     <property name="toolTip">
      <string>This is main setting. Set this to path of a folder with translation files
for your project (or a subproject for your target language).</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="RelPathSaver" name="kcfg_PoBaseDir"/>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="l1">
     <property name="text">
      <string comment="@label:textbox">Root folder:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>poBaseDir</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KComboBox" name="kcfg_LanguageSource">
     <item>
      <property name="text">
       <string>Automatic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Application</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Project</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="kcfg_ProjLangTeam">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="l5">
     <property name="text">
      <string>Language-Team:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LangCodeSaver</class>
   <extends>QLineEdit</extends>
   <header>prefs.h</header>
  </customwidget>
  <customwidget>
   <class>RelPathSaver</class>
   <extends>QLineEdit</extends>
   <header>prefs.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_ProjectID</tabstop>
  <tabstop>LangCode</tabstop>
  <tabstop>kcfg_MailingList</tabstop>
  <tabstop>poBaseDir</tabstop>
  <tabstop>glossaryTbx</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>LangCode</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>kcfg_LangCode</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
